model/vm_host.rb

```diff
@@ -272,4 +272,15 @@ def available_storage_gib
   def total_storage_gib
     storage_devices.sum { _1.total_storage_gib }
   end
+
+  def render_arch(arm64:, x64:)
```

Comment:
Since the rhizome version has default values, should we also include default values to keep consistency across the interfaces?

https://github.com/ubicloud/ubicloud/blob/4dc5c281c9a0c93ad551739d015b91f4aaa8e903/rhizome/common/lib/arch_class.rb#L25

Reply:
I think the defaults are never used in this case, so probably not (fair question: did the original ever use its defaults?)